TweetMate/src/main/java/com/ybsystem/tweetmate/databases/PrefBase.java

```diff
@@ -0,0 +1,28 @@
+package com.ybsystem.tweetmate.databases;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+
+import androidx.preference.PreferenceManager;
+
+import com.ybsystem.tweetmate.R;
+import com.ybsystem.tweetmate.application.TweetMateApp;
+
+public class PrefBase {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1118)